server/src/lib/model/clips.ts

```diff
@@ -70,15 +74,18 @@ interface ClipMetrics {
  * Model for tracking clip information of all files in storage.
  */
 export default class Clips {
+  db: DB;
   allClips: ClipList;
   unverifiedClips: string[];
   verifiedClips: string[];
   metrics: ClipMetrics;
   loaded: boolean;
   randomEngine: Random.MT19937;
+  isMigrated = false;
```

Comment:
for consistency, would be good to declare this here, and then set the default value in the constructor